app/presenters/instance_presenter.rb

```diff
@@ -25,4 +24,10 @@ def status_count
   def domain_count
     Rails.cache.fetch('distinct_domain_count') { Account.distinct.count(:domain) }
   end
+
+  def open_registrations
```

Comment:
I've added some specs but I'm not sure how to mark the fact that I've done that.